cmd/containerd/builtins_cri.go

```diff
@@ -1,5 +1,5 @@
-//go:build (linux && !no_cri) || (windows && !no_cri)
-// +build linux,!no_cri windows,!no_cri
+//go:build !no_cri
+// +build !no_cri
```

Comment:
This change will cause the CRI plugin to also build on FreeBSD.  I don't see any compilation errors with that, but it's certainly functionality that no one is testing right now.  What do you think about adding darwin to the list (`go:build (linux || windows || darwin) && !no_cri`) instead?

Reply:
Opened https://github.com/containerd/project-checks/issues/5 to track the fileheader checker issue.